apps/web/src/pages/Pools/BrowsePoolsPage/BrowsePoolsPage.test.tsx

```diff
@@ -98,23 +104,27 @@ describe("BrowsePoolsPage", () => {
     );
   });
 
-  it("should only show IT jobs", async () => {
+  it("should only show IT and Executive jobs", async () => {
     renderBrowsePoolsPage({
-      pools: [publishedItJobsPool, publishedExecJobsPool],
+      pools: [publishedItJobsPool, publishedExecJobsPool, publishedIAPJobsPool],
```

Comment:
If the pool `publishedIAPJobsPool` is being added, why is the test called **should only show IT and Executive jobs**?

Reply:
Just to verify IAP job is not displayed in here

Reply:
Yeah, the test verifies that it gets filtered out